src/counts.jl

```diff
@@ -29,6 +38,17 @@ function addcounts!(r::AbstractArray, x::IntegerArray, levels::IntUnitRange)
 	return r
 end
 
+"""
+    addcounts!(r::AbstractArray, x::IntegerArray, levels::IntUnitRange, wv::weightVector)
```

Comment:
I think you should merge this with the previous definition, with the following signature:
```julia
addcounts!(r::AbstractArray, x::IntegerArray, levels::IntUnitRange[, wv::weightVector])
```